channeldb/migration21/migration.go

```diff
@@ -184,18 +200,23 @@ func MigrateDatabaseWireMessages(tx kvdb.RwTx) error {
 		}
 	}
 
-	// Next, we'll update all the present close channel summaries as well.
+	return nil
+}
+
+func migrateCloseChanSummaries(tx kvdb.RwTx) error {
+	closedChanBucket := tx.ReadWriteBucket(closedChannelBucket)
+
+	// Exit early if bucket is not found.
+	if closedChannelBucket == nil {
```

Comment:
Change here is that we exit early instead of returning error if bucket is not found.